android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -291,7 +275,7 @@
      * track instead of skipping to the previous track when getting the PREVIOUS
      * command
      */
-    private static final long REWIND_INSTEAD_PREVIOUS_THRESHOLD = 3000;
+    private static final long REWIND_INSTEAD_PREVIOUS_THRESHOLD = 10000;
```

Comment:
10 seconds is a little too much, no? please test this a bit more.
Maybe the right number is in between? 5 seconds?

I suppose it rewinds if the current time is bigger than the number given, otherwise it goes back. 10 seconds seems a bit much. I guess I have to try it with you.

Reply:
That was a little mistake, I changed the value just for testing. It should be 3 seconds.